content/source/docs/enterprise/vcs/troubleshooting.html.md

```diff
@@ -84,3 +84,101 @@ This often happens when the VCS connection has been somehow broken: it might hav
 The domain name for Terraform Enterprise changed on 02/22 at 9AM from atlas.hashicorp.com to app.terraform.io. If the OAuth client was originally configured on the old domain, using it for a new VCS connection can result in this error.
 
 The fix is to update the OAuth Callback URL in your VCS provider to use app.terraform.io instead of atlas.hashicorp.com.
+
+### Debugging Certificate Errors on Private Terraform Enterprise
```

Comment:
Let's drop one of those `#`s to make this a new top-level H2. The rest of the stuff in General is "symptom -> likely problem -> fix," and this is something different that doesn't quite fit with those.

Reply:
Done! 👍